.changeset/violet-meals-work.md

```
@@ -0,0 +1,40 @@
+"@seed-design/tailwind4-theme": minor
+"@seed-design/rootage-artifacts": minor
+"@seed-design/css": minor
+---
+
+시맨틱 stroke 컬러 토큰을 업데이트합니다.
+
+- 이름이 변경되는 stroke 토큰
+
+| 기존              | 신규              | 비고 |
+| ----------------- | ----------------- | ---- |
+| **neutral-muted** | neutral-subtle    |
```

markdownlint-cli2 (0.17.2)

Table column count
Expected: 3; Actual: 2; Too few cells, row will be missing data

(MD056, table-column-count)